bot/kodiak/test_evaluation.py

```diff
@@ -3048,6 +3048,27 @@ def test_get_merge_body_strip_html_comments() -> None:
     assert actual == expected
 
 
+def test_get_merge_body_cut_body_before() -> None:
+    pull_request = create_pull_request()
+    pull_request.body = "hello <!-- testing -->world"
+    actual = get_merge_body(
+        config=V1(
+            version=1,
+            merge=Merge(
+                message=MergeMessage(
+                    body=MergeBodyStyle.pull_request_body,
+                    cut_body_before="<!-- testing -->",
+                )
+            ),
+        ),
+        pull_request=pull_request,
+        merge_method=MergeMethod.squash,
+        commits=[],
+    )
+    expected = MergeBody(merge_method="squash", commit_message="world")
```

Comment:
I think we may want to keep the "<-- testing -->" snippet so that we only remove content before the `cut_body_before` string.

```suggestion
    expected = MergeBody(merge_method="squash", commit_message="<!-- testing -->world")
```

And then users can use the strip_html_comments option to remove the comment if they want.